include/qpid/dispatch/server.h

```diff
@@ -418,6 +418,22 @@ typedef struct qd_server_config_t {
     uint32_t max_frame_size;
 
     /**
+     * The max_sessions value is the number of sessions allowed on the Connection. 
+     * This value minus one is the Open performative channel-max setting.
+     */
+    uint32_t max_sessions;
```

Comment:
What will be do with the value 0?  Should we use channel_max to avoid this question?